proto/service/gribi.proto

```diff
@@ -38,54 +40,54 @@ message ModifyRequest {
 
 // A single request to add/modify/remove a single AFT entry
 message AFTOperation {
-        // Associate an ID with this specific request since this allows the client
-        // to determine exactly which modifications have been completed in the
-        // case that are multiple operations within the AFT.
-        uint64 id = 1;
+  // Associate an ID with this specific request since this allows the client
+  // to determine exactly which modifications have been completed in the
+  // case that are multiple operations within the AFT.
+  uint64 id = 1;
 
-        // Network Instance (aka vrf aka routing-instance) to apply to
-        string network_instance = 2;
+  // Network Instance (aka vrf aka routing-instance) to apply to
+  string network_instance = 2;
 
-        // The operation that this request is - either add, replace, or delete
-        // any entry from the specific AFT.
-        enum Operation {
-          // INVALID indicates that the sender did not set the operation, and
-          // should be considered an error.
-          INVALID = 0;
-          // ADD creates an entry within the table, it should fail if the entry
-          // already exists.
-          ADD = 1;
-          // REPLACE replaces an existing entry within a table, it should fail if
-          // the entry does not exist. A replace operation should contain all of
-          // the relevant fields, such that existing entry is completely replaced
-          // with the specified entry.
-          REPLACE = 2;
-          // DELETE removes an entry from the table, it should fail if the entry
-          // does not exist.
-          DELETE = 3;
-        }
-        Operation op = 3;
+  // The operation that this request is - either add, replace, or delete
+  // any entry from the specific AFT.
+  enum Operation {
+    // INVALID indicates that the sender did not set the operation, and
+    // should be considered an error.
+    INVALID = 0;
+    // ADD creates an entry within the table, it should fail if the entry
+    // already exists.
```

Comment:
We'd updated this comment to indicate that multiple adds are allowed. 
It's fine to update it in a subsequent commit if you'd like to avoid having to regenerate the bindings again

Reply:
Never mind, looks like this diff is a base line update without any new stuff outside the enum paths and adding all the AFT types.

Reply:
ACK -- this is just the update to the existing version. I'm going to be bringing the new changes into the repo in a subsequent PR.